.github/ISSUE_TEMPLATE/feature-request-en.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: "[FEATURE] Short description of the feature"
+labels: ''
+assignees: ''
+
+---
+
+## Required information
+
+**Is your feature request related to a problem? Please describe.**
```

Comment:
```suggestion
**Is your feature request related to a problem? Please describe.**
<!-- Mandatory -->
```

Reply:
Having it in the preceding section header is sufficient.